### DISCUSSION TITLE
wifi issues

### DISCUSSION THREAD

**Discussion by gpiroddi** [rern/rAudio#960] [Fixed Issues]
Hallo,
I had to reinstall rAudio. I flashed a ssd with rAudio-1-64bit-20220526.
Even if wifi is apparently switched on it does not find any network and also disconnects if I reboot.
Previously the rAudio-1-64bit-20220526 was working fine. I also tried updating to 20220703 with same problems.

Any idea?
Thanks
Giovanni

**Comment by gpiroddi** [comment]
Just tried on another Raspberry. Same issue so it shouldn't be a hardware problem.

**Comment by rern** [comment]
Please try today's update.

**Reply by gpiroddi** [reply]
Hi rern,
it works now.
Using another software was very painful  :)
Rebuild of the DB is painful as usual but this seems a problem for all the software that I have tried.
Great work! 
Thanks
Giovanni

**Comment by rojumena** [comment]
Hi,
I am having a similar problem with 202207101 with pi3b. 
  What I have done so far:
     *formatted and etched sd cards a dozen times. 
       *The system boots and loads runeaudio 
        *with lan cable connected I can use ipad to control rune audio 
        *loaded music on usb and enable file sharing and played songs 
However, 
I cannot find a way to update the software or enable wireless, although a scan does display local wireless networks.
 clicking on a wireless network requests a password which accepted and appears to be connecting, then stop me reverts to lan. 
I know i am missing something as I have done this before … age is catching up!!
The last version of runeudio served me well for about two to three years.
Thanks for the great work!

**Reply by rojumena** [reply]
Update!! 
I reformatted my sd card and reflashed 202207101 to start with a fresh install. I see the update button together with the radio software options. The update button is not hi-lited so is it safe to assume I have the latest rAudio version? 
Moving onto enabling wireless settings:
Although I enter SSID and password, SSID is being stored but Password is not being retained. 
I tried scanning and using the scanned source to enable access to my wireless network - no change. I would prefer to have avoided ssh but tried anyway. I can only proceed to password where I am locked out for using and incorrect password for sudo ssh rAudio.local@xx.x.x.xxx

**Reply by rern** [reply]
Try:

[How To](https://github.com/rern/rAudio-1#how-to) > Before power on: > Wi-Fi pre-configure
or
Command: `wifi-menu`

**Reply by rojumena** [reply]
I did try the Wi-Fi pre-configure but the problem persisted. 

However, I narrowed down the source of problem to PBCAC!!!   :-)    Problem Between Chair And Computer!!!! I overlooked one detail...
I just remembered that my router has added security, and that the pi mac address would need to be entered manually......! 
Also, I had to double check that the ethernet mac address, although similar to the  wlan address, is NOT the same and the router software tends to auto complete the mac address based on the first 3 alpha-numeric inputs. For my pi3 the first three pairs of data are identical  but the rest change.
I hope this helps other senior members whose eyes are not as good as they used to be. It's not a good enough excuse :-).
Thanks I love your work after trying the usual suspects. Thank you again!
